incubator/distribution/templates/mongo-connection-secret.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.mongodb.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "distribution.fullname" . }}-mongo-connection
+  labels:
+    app: {{ template "distribution.name" . }}
+    chart: {{ .Chart.Name }}-{{ .Chart.Version }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+type: Opaque
+data:
+  {{- if .Values.mongodb.mongodbUsername }}
+  mongo_connectionString: {{ template "mongodb.url" . }}
+  audit_mongo_connectionString: {{ template "mongodb.audit.url" . }}
```

Comment:
Can probably remove the if/else here as they have the same body.

Reply:
@rjkernick done